sdk/tools/bin2c.c

```diff
@@ -167,8 +167,8 @@ int main(int argc, char* argv[])
 
     /* Generate the header file and close it */
     fprintf(outHFile, "/* This file is autogenerated, do not edit. */\n\n");
-    fprintf(outHFile, "#define %s_SIZE %lu\n"          , argv[5], bufLen);
-    fprintf(outHFile, "extern unsigned char %s[%lu];\n", argv[5], bufLen);
+    fprintf(outHFile, "#define %s_SIZE %lu\n"          , argv[5], (unsigned)bufLen);
+    fprintf(outHFile, "extern unsigned char %s[%lu];\n", argv[5], (unsigned)bufLen);
```

Comment:
Can we use `unsigned int` written explicitly instead?

Reply:
Done, it's unsigned long though.

Reply:
ok, then "unsigned long" instead ^^